src/controllers/api.controller.ts

```diff
@@ -470,7 +466,9 @@ export const claimReport = async (
 
     const claimsOfSubj = await prisma.$queryRaw<ReportI>`
       ${Prisma.raw(baseQuery)}
-      WHERE c."subject" = ${claim.subject} AND c."id" != ${Number(claimId)}
+      WHERE c."subject" = ${claim.subject.toLocaleLowerCase()} AND c."id" != ${Number(
```

Comment:
do we need to maybe lowercase both sides? do we know that c.subject will be already lower case?

Reply:
no i recived it in PascalCase like 'VoiceOfChildren' but in related ones 'voiceofchildren'